src/Campaigns/ServiceProvider.php

```diff
@@ -233,10 +233,11 @@ private function registerCampaignBlocks()
      */
     private function loadCampaignOptions()
     {
-        Hooks::addAction('init', LoadCampaignOptions::class);
+        Hooks::addAction('admin_init', LoadCampaignOptions::class);
```

Comment:
@pauloiankoski If these are specific to loading scripts shouldn't we be using `admin_enqueue_scripts`?